widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -183,6 +187,8 @@ export function Quote(props: QuoteProps) {
               USD_VALUE_MIN_DECIMALS,
               USD_VALUE_MAX_DECIMALS
             ),
+            realValue: swap.toAmount,
+            realUsdValue: (swap.to.usdPrice ?? 0) * parseFloat(swap.toAmount),
```

Comment:
Using BigNumber.js is recommended for precision and consistency.

Reply:
fixed.